src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,53 @@
-import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type MovieCardProps = {
+  card: Movie;
```

Comment:
prop name should be movie (i think the tests are not passed because of this name of prop)